plugin/lsp.vim

```diff
@@ -14,6 +14,10 @@ def g:LspOptionsSet(opts: dict<any>)
   options.OptionsSet(opts)
 enddef
 
+def g:LspOptionsGet(key: string): any
```

Comment:
What is use case for this function?  Can you update doc/lsp.txt with a description of this function and how to use it?

Reply:
the usage mostly is for coding, since it cannot directly access lsp#options#lspOptions in user func.